src/expectRevert.js

```diff
@@ -0,0 +1,59 @@
+const { web3 } = require('./setup');
+
+const { expect } = require('chai');
+const colors = require('ansi-colors');
+const semver = require('semver');
+
+async function expectException (promise, expectedErrors) {
+  try {
+    await promise;
+  } catch (error) {
+    for (const expectedError of expectedErrors) {
+      expect(error.message).to.include(expectedError, `Wrong failure type, expected '${expectedError}'`);
+    }
+    return;
+  }
+
+  throw Error('Expected failure not received');
+}
+
+const expectRevert = async function (promise, expectedError) {
+  if (!expectedError) {
+    promise.catch(() => { });
+    throw Error('No revert reason specified: call expectRevert with the reason string, or use expectRevert.unspecified\
```

Comment:
What did you attempt to do here with the newline? There are four spaces at the beginning of next line. Were they on purpose?

Reply:
'attempt to do' :joy: I simply wanted to not go over the max line length, it was my understanding all whitespace at the beginning is ignored.

Reply:
No, it's not! In `configure.js` I added spaces because I wanted them in the output.

https://github.com/OpenZeppelin/openzeppelin-test-helpers/blob/401e0bd10d3bb0e94e423a51454cc0f88694d67c/configure.js#L19-L26